zerver/openapi/zulip.yaml

```diff
@@ -6756,37 +6756,55 @@ paths:
                           Present if `update_display_settings` is present in `fetch_event_types`.
 
                           The color scheme selected by the user.
+
+                          See [PATCH /update-display-settings](/api/update-display-settings)
+                          for api details on display settings.
```

Comment:
We have a standard format for linking to other API endpoints, and it's this:

```
diff --git a/zerver/openapi/zulip.yaml b/zerver/openapi/zulip.yaml
index a7d6fad357..23ca5b6cb4 100644
--- a/zerver/openapi/zulip.yaml
+++ b/zerver/openapi/zulip.yaml
@@ -7556,7 +7556,7 @@ paths:
 
                           The color scheme selected by the user.
 
-                          See [PATCH /update-display-settings](/api/update-display-settings)
+                          See [PATCH /settings/display](/api/update-display-settings)
                           for api details on display settings.
                       default_language:
                         type: string
```

I kinda want to extend the markdown process so that we can write something like `[update-display-settings]` and have the right thing happen (we have the data to do that) for these links.

Ideally such a system would also let us do `[feature-level-49]` and auto-generated the text with a link to the API changelog as well.  

@orientor @MSurfer20 what do you think of this idea?  I'm not sure what syntax would make sense, but it does feel like it could simplify the experience for adding links consistently.

Reply:
I'll update it if you are not looking at it right now.

Reply:
For this PR, I decided this was close enough to finishing that it'd be better for me to just do the quick find-replace.

But I'd appreciate feedback on the markdown feature; we write a lot of these links, especially in the API changelog, and they're kinda fiddly, so it'd be nice if we could remove the fiddliness and need for contributors to be consistent with how we do the labels (etc.).

Can one of you start a thread on #documentation about this idea?

Reply:
(For reference, pasting here the link to the chat topic opened by Purushottam 🙂 )
https://chat.zulip.org/#narrow/stream/19-documentation/topic/Link.20.2Fhelp.20page.20using.20template-name